### DISCUSSION TITLE
How to check if an array is superset of another

### DISCUSSION THREAD

**Discussion by kartik-579** [antonmedv/expr#372] [Q&A]
I have an usecase where i want to check that my input array has all the elements of a given array (superset checking). I checked the docs but am still unable to find the solution to this. Can someone please help me with this?

**Comment by mdmcconnell** [comment]
One possibility is you could check that all of your subset is in the superset using a [predicate expression](https://expr.medv.io/docs/Language-Definition#predicate), like
`"all([1,2,3], {# in [1,2,3,4,5]})"`
https://go.dev/play/p/R-BQbCTCcTt
